src/components/Header/Contact/Contact.css

```diff
@@ -1,19 +1,17 @@
 .contact {
-    display: flex;
```

Comment:
Why 2 spaces instead of 4? Is this a standard for CSS? Do you use any linting tool? If yes, we can add it to the CI of the project